.golangci.yml

```diff
@@ -6,10 +6,10 @@ run:
     - mempool
     - state/indexer
     - state/txindex
+    - third_party
```

Comment:
ignoring `third_party` from lint to reduce diff created by addressing lint errors.